src/main/java/donna/Ui.java

```diff
@@ -0,0 +1,173 @@
+package donna;
+
+import donna.task.Task;
+import donna.task.TaskList;
+
+import java.util.List;
+
+/**
+ * Contains user interface methods for interacting with the application.
+ * Handles displaying messages, task lists, and error information to the user.
+ */
+public class Ui {
+    private boolean wasDataLoaded;
+
+    public Ui(boolean wasDataLoaded) {
+        this.wasDataLoaded = wasDataLoaded;
+    }
+
+    static private String dashedLine() {
+        return "____________________________________________________________________\n";
+    }
+
+    static private String donnaLogo() {
+        return " __      \n"
+                + "|  _  \\  __  _ __   _ _   __  \n"
+                + "|  |  |  | / _ \\| '_  \\ | '_ \\ / _` |\n"
+                + "|  |_| |  (_) |  | |  |  |  |   | (_|  |\n"
+                + "|___/ \\__/|_| |_|_|  |_|\\_,_|\n";
+    }
+
+
+    /**
+     * Returns a greeting message to the user.
+     *
+     * @return The greeting message.
+     */
+    public String getGreeting() {
+        StringBuilder greeting = new StringBuilder();
+//        greeting.append(donnaLogo());
```

Comment:
I think if you're not gonna use this you can delete it. Otherwise I think it should be in line with the rest of the code.
https://se-education.org/guides/conventions/java/intermediate.html#comments

Reply:
You're right. I left it for the text UI, but I can remove it now.